src/core/OmniInputStories.ts

```diff
@@ -156,6 +157,42 @@ export const SuffixStory = <T extends HTMLElement, U extends BaseArgs>(tagName:
     return Suffix;
 };
 
+export const ClearableStory = <T extends HTMLElement, U extends BaseArgs>(
+    tagName: string,
+    inputValue: string | number | string[] = 'The input value'
+) => {
+    const Clearable: ComponentStoryFormat<U> = {
+        render: (args: U) =>
+            html`${unsafeHTML(
+                `<${tagName} data-testid="test-field" label="${ifNotEmpty(args.label)}" value="${args.value}" clearable=${
```

Comment:
Consider using a Boolean attribute for clearable.

Reply:
Updated as per suggestion